fiftyone/utils/labels.py

```
@@ -207,16 +211,31 @@
+            if isinstance(label, fol.Segmentation):
+                if label.mask is not None:
+                    outpath = filename_maker.get_output_path(
+                        image.filepath, output_ext=".png"
+                    )
+                    label.export_mask(outpath, update=update)
+            elif isinstance(label, fol.Detection):
                 if label.mask is not None:
+                    outpath = filename_maker.get_output_path(
+                        image.filepath, output_ext=".png"
+                    )
                     label.export_mask(outpath, update=update)
```

Ruff (0.8.2)

Combine `if` branches using logical `or` operator

Combine `if` branches

(SIM114)